new-pipeline/src/dashboards.js

```diff
@@ -1048,8 +1048,12 @@ function updateOSs() {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = "https://georgf.github.io/fx-data-explorer/index.html?search=" + metric + "&searchtype=in_name&optout=false&channel=" + channel + "&constraint=is_in&version=any"
-  var imgUrl = "https://upload.wikimedia.org/wikipedia/commons/6/64/Icon_External_Link.png"
   var descr = description === null ? metric : description
 
-  return " <a href=\"" + metricUrl + "\" target=\"_blank\">" + descr + "<img src=\"" + imgUrl + "\"/></a>"
+  var div = document.createElement('div');
+  div.classList.add("text-center");
+  div.classList.add("py-5");
+  div.innerHTML =  descr + " <a href=\"" + metricUrl + "\" target=\"_blank\">" + "<i class=\"btn btn-outline-primary fa fa-info-circle\" style='float: right; color: black' aria-hidden=\"true\">  more details </i></a>"
```

Comment:
Visually this looks pretty close.
Some things:
- Use "More details" vs. "more details"
- I don't think the separate `<i>` element is needed, you can just style the `<a>` element.
- This line is pretty long, let's break it up.
- Let's drop the `float: right`, it looks off and makes the button hard to find.

Reply:
`<i>` element is needed for the icon of `More Details` otherwise if you don't need it I'll remove it from there :)

`float:right` helps to keep the `More Details` part to the right. Else it comes just after the description which doesn't look good.